src/models/spaces/populations/mouse-model/mice.ts

```diff
@@ -35,6 +35,8 @@ const MouseGeneticSpec = {
   }
 };
 
+export type MouseColors = "white" | "tan" | "brown";
```

Comment:
Might be a nit, but it would be nice to use the enum from `mouse.ts` if possible. Possibly with a rename from `ColorType` to `MouseColor` which I agree is better/more specific.

Reply:
Ah, true. The `mouse.ts` model didn't exist when I started on the tan mice. I think maybe we're still going to need some work to see how to share/merge differing models of mice (e.g. the `Mouse` that gets defined here is something that extends `Populations.BasicAnimal`, and we don't want that in our backpack, at least not directly, as we've discussed).

In any case, yes it could use the same type. I'll try to get back to it after I'm done with populations.js updates.